OJP/OJP_FareSupport.xsd

```diff
@@ -446,11 +460,18 @@
 					<xs:documentation>URL for booking.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="BookingNote" type="InternationalTextStructure" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Note about booking the FLEXIBLE LINE.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
+			<xs:choice>
+				<xs:element name="BookingNote" type="BookingNoteType" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>Note about booking the (FLEXIBLE) LINE (depreciated).</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element name="BookingNotes" type="BookingNoteStructure" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>Notes about booking the (FLEXIBLE) LINE.</xs:documentation>
```

Comment:
This sounds like that the notes could only be used for a flexible line. I don't understand why this should be, but I would hesitate using them for any line.